wsmaster/che-core-api-system/src/test/java/org/eclipse/che/api/system/server/SystemManagerTest.java

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2017 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.system.server;
+
+import org.eclipse.che.api.core.ConflictException;
+import org.eclipse.che.api.core.notification.EventService;
+import org.eclipse.che.api.system.shared.SystemStatus;
+import org.eclipse.che.api.system.shared.dto.event.SystemStatusEvent;
+import org.eclipse.che.api.workspace.server.WorkspaceManager;
+import org.eclipse.che.dto.server.DtoFactory;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.testng.MockitoTestNGListener;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+
+import java.util.Iterator;
+
+import static org.mockito.Mockito.atLeastOnce;
+import static org.mockito.Mockito.timeout;
+import static org.mockito.Mockito.verify;
+import static org.testng.Assert.assertEquals;
+
+/**
+ * Tests {@link SystemManager}.
+ *
+ * @author Yevhenii Voevodin
+ */
+@Listeners(MockitoTestNGListener.class)
+public class SystemManagerTest {
+
+    @Mock
+    private WorkspaceManager wsManager;
+
+    @Mock
+    private EventService eventService;
+
+    @Captor
+    private ArgumentCaptor<SystemStatusEvent> eventsCaptor;
+
+    private SystemManager systemManager;
+
+    @BeforeMethod
+    public void init() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
If i don't do it explicitly i start getting errors like this:
```
org.mockito.exceptions.verification.TooManyActualInvocations: 
runtimes.stop(<any>);
Wanted 1 time:
-> at org.eclipse.che.api.workspace.server.WorkspaceManagerTest.shouldStopWorkspaceEventIfSnapshotCreationFailed(WorkspaceManagerTest.java:509)
But was 11 times. Undesired invocation:
-> at org.eclipse.che.api.workspace.server.WorkspaceManager.lambda$stopAsync$3(WorkspaceManager.java:813)
```

Which probably means that mockitong doesn't do it's job well, do i miss something ?